app/models/lettings_log.rb

```diff
@@ -418,27 +418,13 @@ def is_dpo
     created_by&.is_dpo
   end
 
-  delegate :service_name, :sensitive, :registered_under_care_act, :primary_client_group, :has_other_client_group, :secondary_client_group, :owning_organisation, :managing_organisation, :support_type, :intended_stay, :created_at, prefix: "scheme", to: :scheme, allow_nil: true
-  delegate :scheme_type, to: :scheme, allow_nil: true
-
   def scheme_code
     scheme&.id ? "S#{scheme.id}" : nil
   end
 
-  def scheme_owning_organisation_name
-    scheme_owning_organisation&.name
-  end
-
-  delegate :postcode, :name, :units, :type_of_unit, :mobility_type, :startdate, prefix: "location", to: :location, allow_nil: true
-  delegate :location_admin_district, to: :location, allow_nil: true
-
-  # This is not the location_code in the db, location.id is just called code in the UI
-  def location_code
-    location&.id
-  end
```

Comment:
Are we sure we're not using this somewhere else?

Reply:
I remember cmd-shift-effing these things at the time so I am confident (never sure with the ol' meta-programming)